packages/components/tests/integration/components/hds/flyout/index-test.js

```diff
@@ -80,10 +80,10 @@ module('Integration | Component | hds/flyout/index', function (hooks) {
             <F.Header>Title</F.Header>
           </Hds::Flyout>`
     );
-    assert.dom('.hds-flyout__title').exists();
-    assert.dom('.hds-flyout__title').hasText('Title');
-    assert.dom('.hds-flyout__icon').doesNotExist();
-    assert.dom('.hds-flyout__tagline').doesNotExist();
+    assert.dom('.hds-dialog-primitive__title').exists();
```

Comment:
I am not convinced of this change. I'll think about it more and review the integration tests when the PR is ready for final review